src/hpr_routing.erl

```diff
@@ -97,95 +103,159 @@ find_routes({uplink, {_Type, DevAddr}}, PacketUp) ->
     ok = hpr_metrics:observe_find_routes(Time),
     Results.
 
+-spec start_crawl_routing() -> pid().
+start_crawl_routing() ->
```

Comment:
if this process dies is there anything that will re-start the recursion cycle or is it a problem if that _doesn't_ happen?

Reply:
The table belongs to the supervisor, if the table is not present this process will be reset.
But that leaves a chance that it comes back through the sleep, and now there's more crawls happening.
Which is not the worst since they're all doing the same sweep. But too many could bog down the table. 

hmmm, could probably spawn_link?
I'd have to rethink what it means to "cancel" the crawling so it doesn't kill the supervisor.
But it would keep the the supervisor restarting a bunch of times from making a fleet of these procs.

Reply:
It's been moved to a gen_server `hpr_routing_cache`.